template/9-design.md

```diff
@@ -2,23 +2,34 @@
 
+### Essential screens
+
+**Login page**: First screen that appears when the user is not authenticated. It allows the user to authenticate and access the application.
+
+**Feed**: The feed is the central screen of the application. Shown when starting the application as an authenticated user, it displays the posts that are at less than one hundred meters away from the user. This screen allows him to sort the feed by hottest, top, latest, or even nearest posts. The feed displays, for every post, its title, the first seven lines of its description, its owner, the number of comments, the date of the post along with its upvote score. It also allows the user to upvote or downvote any posts directly from the feed screen.
+
+**Profile page**: The profile screens displays all the information associated to the user, such as his username, display name, number of centauri points, badges, number of followers along with his number of followed users. Moreover, it displays the title and description of all of his posts along with all of his comments. The user is able to delete any post or comment that he created, from anywhere, directly from the profile page.
+
+**Post creation page**: Screen allowing the user to write a new post by setting its title and description. Before posting, the user is able to choose a group of nearby users with whom he wants to post. He can also decide whether to send a notification to his followers.
+
+**Post page**: Page displaying all the information associated to a particular post. This screen is displaying the title, complete description, owner, date, distance, upvote score, and all the comments associated with the post. The user is able to sort the comments by top or latest at his convenience, and he can follow up on any comment, creating a thread of comments to discuss and interact about the original post.
+
+**Challenge page**: The challenge page displays all the current available challenges that the user can complete alone or as a group. A challenge is described by the post title and distance, along with the time left to complete it. By clicking on a challenge, we are redirected to the map, allowing the user to localize it more easily.
+
+**Group page**: The group page allow the user to create or join a group with nearby users. It allow them post and complete challenges together.
+
+**Map page**: The map page enables the user to switch between multiple maps that are of two main types. 
+First, we have the maps, allowing the user to see the precise location of his posts and current challenges. Moreover, he is also able to display all the posts that are less than one hundred meter away.
+Additionally, the user is also able to access the heatmap, enabling him to see places around the world that are currently attracting a lot of interaction.
```

Comment:
"that are currently trending"

Reply:
Thank you !

Fixed in: [615314c](https://github.com/ProximaEPFL/prd/pull/8/commits/615314c78843abeb9be39f8874b637ee9a70005a)